networkx/drawing/nx_pylab.py

```diff
@@ -353,9 +353,11 @@ def draw_networkx_nodes(
         mapped to colors using the cmap and vmin,vmax parameters. See
         matplotlib.scatter for more details.
 
-    node_shape :  string (default='o')
+    node_shape :  string or array of strings (default='o')
         The shape of the node.  Specification is as matplotlib.scatter
-        marker, one of 'so^>v<dph8'.
+        marker, one of 'so^>v<dph8'. This can be a single shape, in which case
+        it will be applied to all nodes. Otherwise if it is an array, the 
```

Comment:
```suggestion
        it will be applied to all nodes. Otherwise if it is an array, the
```
Trailing whitespace.